src/pipes/mongo-datestart.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  HttpException,
+  HttpStatus,
+  Injectable,
+  PipeTransform,
+} from '@nestjs/common';
+
+@Injectable()
+export class ParseMongoDateStartPipe implements PipeTransform {
+  transform(value: number) {
+    try {
+      const hour = 86400;
+      const new_value = value - (value % hour);
+      const mul = 1000;
+      return new Date(new_value * mul);
```

Comment:
This works but it seems so weird. You might wanna use `setHours` to make it look cleaner. Anyway, it's optional.

Reply:
i go look for sethour() it set just hour if i want 00:00:00 sethour(0),setminute(0),setsecond(0) right?

Reply:
ok i get it

Reply:
discuss golf la he said front gonna sent 00.00.00